libs/sdk-core/src/breez_services.rs

```diff
@@ -930,6 +919,106 @@ impl BreezServices {
         Ok(self.btc_receive_swapper.refund_swap(req).await?)
     }
 
+    pub async fn onchain_payment_limits(&self) -> SdkResult<OnchainPaymentLimitsResponse> {
```

Comment:
Can we add `fetch` prefix to this function name?

Reply:
@roeierez I agree it's a bit long, 5 letter longer then the function it supersedes, but as far as I'm aware we have `fetch` prefixes on almost all functions that makes a network request.

Reply:
I don't think it is the case right now. A few examples: `open_channel_fee`, `lsp_info`, `recommended_fees`